Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7.9-slim-buster as base
+
+ENV PATH /usr/local/bin:$PATH
+ENV VIRTUAL_ENV "/venv"
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH "$VIRTUAL_ENV/bin:$PATH"
+WORKDIR /app
+RUN pip install poetry
+EXPOSE 5000
+COPY poetry.lock pyproject.toml /app/
+RUN poetry install --no-root
+COPY . /app
```

Comment:
If you move some packages into dev-dependencies in pyproject.toml (see my other comment), then you can run `poetry install --no-dev` to skip them.

So you can have a smaller poetry installation for the `production image` if you move this and have different install commands in production/development stages.